Blaise-Shyaka/My-Mentor
# [finishes #170345230] complete user sign up form

# What does this PR do?

It contains the sign-up page

# Tasks to be completed

- Sign up form that captures user information such as email, bio, and password.
- Add in-form validation
- Make it responsive

# How should this be manually tested?

- Check into the views directory and open sign-up.html. 
- Resize the browser to verify its responsiveness
- Try to submit the form without filling important areas.
